oteapi/models/mappingconfig.py

```diff
@@ -1,27 +1,28 @@
 """Pydantic Mapping Configuration Data Model."""
-from typing import Annotated, Dict, List, Optional
+from typing import Dict, List, Optional, Tuple
 
-from pydantic import Field, conlist
+from pydantic import Field
 
 from oteapi.models.genericconfig import GenericConfig
 
-SemanticTriple = Annotated[list, conlist(str, min_items=3, max_items=3)]
-
 
 class MappingConfig(GenericConfig):
     """Mapping Strategy Data Configuration."""
 
     mappingType: str = Field(
-        ..., description="Type of registered mapping strategy. E.g., `mapping/demo`."
+        ...,
+        description="Mapping type.  Normally this is just 'triples'.",
     )
     prefixes: Optional[Dict[str, str]] = Field(
-        None,
+        {},
         description=(
             "List of shortnames that expands to an IRI "
             "given as local value/IRI-expansion-pairs."
         ),
     )
-    triples: Optional[List[SemanticTriple]] = Field(
-        None,
-        description="List of semantic triples given as (subject, predicate, object).",
+    triples: Optional[List[Tuple[str, str, str]]] = Field(
+        [],
```

Comment:
The same is true here as for the `Optional[dict]` above.

Reply:
Please recreate `SemanticTriple` in your image. I.e., `RDFTriple = Tuple[str, str, str]` or similar.

Reply:
ok